packages/liveblocks-client/src/room.ts

```diff
@@ -1064,38 +1085,47 @@ function makeStateMachine<
     batchUpdates(() => {
       for (const message of messages) {
         switch (message.type) {
+          // Read event
```

Comment:
Do we really have a notion of "read" or "write" for messages received from the server?